linz2osm/convert/osm.py

```diff
@@ -38,15 +38,33 @@ def get_srtext_from_srid(srid):
 
 def get_layer_geometry_type(database_id, layer):
     """ Returns a (geometry_type, srid) tuple """
-    cursor = connections[database_id].cursor()
-    cursor.execute('SELECT type,srid FROM geometry_columns WHERE f_table_name=%s;', [layer.name])
-    r = cursor.fetchone()
-    return tuple(r[:2])
+    if layer.geometry_type == 'RELATION':
+        geotypes = get_relation_geometry_types(database_id, layer)
+        if not geotypes:
+            raise ValueError("No members for relation")
+        srid = geotypes.values()[0][1]
+        if not all([srid == s for (geotype, s) in geotypes.values()]):
+            raise ValueError("SRIDs differ for relation layers!")
+        return ("GEOMETRYCOLLECTION", srid)
```

Comment:
Do we need to check they're all the same if the return value is always a GeometryCollection? alt: should it always be a GeometryCollection rather than the relation layer geotype?